packages/node/src/requestdata.ts

```diff
@@ -310,8 +310,8 @@ function extractQueryParams(req: PolymorphicRequest): string | Record<string, un
 
   return (
```

Comment:
On node v8 `URL` would have always been `undefined` whereas now it'll parse the url correctly.